docs/writing-a-good-changelog-entry.md

```diff
@@ -0,0 +1,35 @@
+# Writing a good Changelog Entry
+
+Part of our standard PR template is "Proposed changelog entry for your changes". This document will help you write a good one.
+
+## Do I even need one?
+
+Our changelog is intended for the users of Jetpack, primarily end users but also third-party developers who use our public APIs and packages.
+
+If you've changed something that users will notice, like improving the display of a block or fixing a bug that prevents a site feature from working, then by all means suggest a changelog entry.
+
+If you've improved our CI or development environments, or refactored old code without user-visible changes, or something like that, it's probably not something that's worth mentioning in the changelog. But don't worry, we still appreciate it!
```

Comment:
I wonder if we should suggest writing something like "N/A", or "none needed", to make it clear that you've considered whether a changelog entry was needed and decided against it, and not just ignored the prompt.

Reply:
ok